certbot/main.py

```diff
@@ -743,8 +743,14 @@ def main(cli_args=sys.argv[1:]):
     config = configuration.NamespaceConfig(args)
     zope.component.provideUtility(config)
 
-    log.post_arg_parse_setup(config)
-    make_or_verify_needed_dirs(config)
+    try:
+        log.post_arg_parse_setup(config)
+        make_or_verify_needed_dirs(config)
+    except errors.Error:
+        # Let plugins_cmd be run as un-privileged user.
+        if config.func != plugins_cmd:
```

Comment:
I don't see plugins_cmd defined anywhere - have you tested this code?
Also it would be good to write tests for this.

Reply:
@SwartzCr, `plugins_cmd` function is defined in `certbot.main`.

I've added a test, see if it is ok.